terraform/ec2/linux/main.tf

```diff
@@ -130,10 +130,9 @@ resource "null_resource" "integration_test" {
 
 data "aws_ami" "latest" {
   most_recent = true
-  owners      = ["self", "506463145083"]
```

Comment:
Delete the owners since the name of the AMI is unique enough and for some case such as MacOS or other non buildable with EC2 Image Builder, we cannot search the required AMI.

Reply:
Does this still work on forks after making this change?

Reply:
Yes. It is public API. Therefore, it can still find the query.